webservice/export-catalogs.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var  ensureDb = require('dbjs/valid-dbjs')
+  , memoize = require('memoizee');
+
+module.exports = memoize(function (db) {
+	// ensure db
+	ensureDb(db);
+
+	var catalogs = [];
+	// all enums
+	var enums = db.StringLine.extensions.toArray().filter(function (extension) {
+		return extension.meta && extension.members;
+	}).map(function (extension) {
+		var catalogItem = { name: extension.__id__ };
+		catalogItem.options = extension.members.toArray().map(function (option) {
+			return { name: option, label: extension.meta[option].label };
+		});
+		return catalogItem;
+	});
+	[].push.apply(catalogs, enums);
```

Comment:
@roxanadina maybe it does not bring much semantically, but still @mtuchowski has a point here. It's like we use namespace for our important groups (`services`, `documents` etc.) while flood the catalogs with all others directly. In the worst case scenario we can hit a name collision, when I for example have an enum entry `services`. So we should group them at least for the purpose of avoiding that (still there will be possibility of name collision between enum entries themselves).